File: TextosJapones/src/main/java/org/jisho/textosJapones/model/enums/Tela.java
package org.jisho.textosJapones.model.enums;

public enum Tela {
	
	TEXTO("Texto"), LEGENDA("Legenda"), MANGA("Manga");

	private String tela;

	Tela(String tela) {
		this.tela = tela;
	}

	public String getDescricao() {
		return tela;
	}
	
	// Necessário para que a escrita do combo seja Ativo e não ATIVO (nome do enum)
	@Override
	public String toString() {
		return this.tela;
	}


}

File: ProcessaTexto/src/main/kotlin/br/com/fenix/processatexto/model/enums/Tela.kt
package br.com.fenix.processatexto.model.enums

enum class Tela(val descricao: String) {
    TEXTO("Texto"), LEGENDA("Legenda"), MANGA("Manga");

    // Necessário para que a escrita do combo seja Ativo e não ATIVO (nome do enum)
    @Override
    override fun toString(): String {
        return descricao
    }
}